packages/openops/src/lib/ai/providers.ts

```diff
@@ -137,3 +129,23 @@ const invalidConfigError = (
     error: { errorName, errorMessage },
   };
 };
+
+function sanitizeProviderSettings(
+  settings: Record<string, unknown> | null | undefined,
+): Record<string, unknown> {
+  if (!settings) return {};
+
+  const sanitized: Record<string, unknown> = {};
+
+  for (const [key, value] of Object.entries(settings)) {
```

Comment:
why is this relevant?
is there an error if we pass settings that are null/undefined/empty?

Reply:
hmm not sure, I've inherited it from @MarceloRGonc